BethREADME.md

```diff
@@ -0,0 +1,61 @@
+COMPLETED STEPS:
+
+Setup code, bundle install
+Rspec, update Spec Helper
+
+Look for folders and empty files
+spec folder, features, also app.rb already there
+
+```
+USER STORY 1:
+As a marketeer
+So that I can see my name in lights
+I would like to register my name before playing an online game
+```
+
+Completed feature tests for landing page, form for player name.
+Next page displays the player's name.
+
+```
+“the marketeer will be presented the choices (rock, paper and scissors)”
+```
+
+Added new erb view files for move_rock, move_paper and move_scissors
+Added jpeg of rock, paper, scissors
+
+
+NEED TO DO:
```

Comment:
Just seen these - sounds like a good plan of action! 

Having a `Game` class will make this much easier. You could test drive the `Game` class without even considering the webapp, i.e. having methods like `decide_computer_move`, which randomly picks a move for the computer, and then a method like `decide_winner`, which you could pass two arguments (player_move, computer_move), and then that applies the logic of who the winner is. Once you've got a working class, it will be much easier to put that into use in the `app`, and I think tackling the problem that way (rather than thinking firstly how the website would implement it) is probably easier (in my opinion!).